src/System.Collections.Concurrent/tests/ProducerConsumerCollectionTests.cs

```diff
@@ -731,25 +731,19 @@ public void ManyConcurrentAddsTakes_EnsureTrackedCountsMatchResultingCollection(
         }
 
         [Theory]
-        [InlineData(ConcurrencyTestSeconds)]
-        public void ManyConcurrentAddsTakes_CollectionRemainsConsistent(double seconds)
+        [InlineData(1000)]
```

Comment:
I set the number to 3000000, which on my machine runs for about the same time as the old test. I also moved it to Outerloop, since this test library is running too long for innerloop (~15s) and this test accounts for ~5s of that.